test/unit/remote_control_test.lua

```diff
@@ -813,3 +813,24 @@ function g.test_reconnect()
         })
     end)
 end
+
+function g.test_gc()
+    local ok, err = remote_control.bind('127.0.0.1', 13301)
+    t.assert_equals(err, nil)
+    t.assert_equals(ok, true)
+
+    local RLIMIT_NOFILE = tonumber(io.popen('ulimit -n'):read('*all'))
+    require('log').info('RLIMIT_NOFILE: %d', RLIMIT_NOFILE)
```

Comment:
Seems this is a debug line

Reply:
Well yes, but actually no. (But yes).
Anyway, it's helpful in CI.

Reply:
Yet ulimit didn't work in CI, so I refactored the test